recipe/spyder-menu.json

```
@@ -86,11 +36,6 @@
                     "StartupWMClass": "Spyder",
                     "MimeType": [
                         "text/x-python",
```

```suggestion
                        "text/x-python"
```
Final list item with "," results in `JSONDecodeError`